content/roster/2023/_index.md

```diff
@@ -3,9 +3,19 @@ title: "2023 Roster"
 slug: "2023"
 layout: roster
 draft: false
-showpreseason: true
-showroster: false
-showpractice: false
+showpreseason: false
+showcaptains: true
+showcoach: true
+showroster: true
+showpractice: true
 ---
 
+#### Captain:
```

Comment:
these changes don't do anything (this stuff is the "content" of the _index page but I'm not actually using it in the rendering)

please delete from #### Captain: onwards